core/src/main/java/com/adobe/aem/dot/dispatcher/core/parser/ConfigurationReader.java

```diff
@@ -260,33 +260,24 @@ public void advancePastThisElement() {
     ConfigurationValue<?> startToken;
 
     // Clear the "{"
-    try {
-      startToken = this.next(false);
-    } catch(ConfigurationSyntaxException csEx) {
-      // Since `isNextChar()` succeeded, this should never happen.  If it does, leave reader pointer untouched.
-      return;
-    }
+    startToken = this.next(false);
```

Comment:
It seems like `next()` throws `ConfigurationSyntaxException` so don't we have to handle it?  Or doesn't `advancePastThisElement()` have to `throws` it?
```
  public ConfigurationValue<String> next() throws ConfigurationSyntaxException {
    return this.next(false);
  }
```